src/template.js

```diff
@@ -8,6 +8,8 @@ const makeInteger = require('makeInteger');
 
 const TEMPLATE_VERSION = '1.1.0';
 const INSIGHTS_OBJECT_NAME = 'AlgoliaAnalyticsObject';
+const INSIGHTS_LIBRARY_URL =
+  'https://cdn.jsdelivr.net/npm/search-insights@2.0.4';
```

Comment:
How does this template get updated? Is it something we submit or is it up to the customer to do anything?